hieradata/dev/wso2/wso2is/pattern-5/default.yaml

```diff
@@ -0,0 +1,512 @@
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+
+# Properties in identity.xml
+wso2::session_persist:
+  enabled: true
+  temporary: true
+  pool_size: 0
+
+wso2::session_cleanUp:
+  enabled: true
+  timeout: true
+  period: 1140
+
+wso2::operation_cleanUp:
+  enabled: true
+  period: 720
+
+wso2::openID:
+  skip_user_consent: false
+  remember_me_timeout: 7200
+  disable_dumbmode   : false
+
+wso2::time_config:
+  session_idle_timeout : 15,
+  remember_me_timeout  : 20160,
+
+
+wso2::oAuth:
+  app_info_cache_timeout : -1
+  auth_grant_cache_timeout  : -1
+  session_data_cache_timeout   : -1
+  claim_cache_timeout   : -1
+  auth_code_default_validity_period : 300
+  access_token_default_validity_period : 3600
+  user_access_token_default_validity_period : 3600,
+  refresh_token_validity_period : 84600,
+  time_stamp_skew   : 300
+  enable_OAuth_cache : false
+  renew_refresh_token_for_refresh_grant : true
+  strict_client_credential_validation   : false
+  enable_assertions_UserName : false
+  enable_access_token_partitioning : false
+  auth_context_token_generation_enabled : false
+  auth_context_token_generation_TTL : 15
+  id_token_expiration : 3600
+  openIDConnect_skip_user_consent: false
+
+wso2::sso:
+  persistance_cache_timeout: 157680000
+  session_index_timeout: 157680000
+  single_logout_retry_count: 5
+  single_logout_retry_interval: 60000
+  sAMLresponse_validity_period: 5
+  use_authenticated_user_domain_crypto: false
+  slo_host_name_verifi_enabled: true
+
+wso2::passiveSTS:
+  slo_host_name_verifi_enabled: true
+
+wso2::evenet_listeners:
+  user_store_action_listener: true
+  identity_mgt_event_listener: false
+  identity_governance_event_listener: true
+  scimuser_operation_listener: true
+  identity_store_event_listener: true
+  daslogin_data_publisher: false
+  dassession_data_publisher: false
+  authn_data_publisher_proxy: true
+
+wso2::cache:
+  appauth_framework_session_context: false
+  authentication_context_cache    : false
+  appauth_framework_session_contex: false
+  authentication_request_cache : false
+  app_info_cache : false
+  authorization_grant_cache : false
+  oauth_cache : false
+  oauth_session_data_cache : false
+  samalsso_participant_cache : false
+  samalsso_session_index_cache : false
+  samalsso_session_data_cache : false
+  service_provider_cache: true
+  provisioning_connector_cache : true
+  provisioning_entity_cache : false
+  service_provider_provisioning_connector_cache: false
+  idP_cache_byauth_property : true
+  idp_cache_byhri : true
+  idp_cache_by_name: true
+
+
+
+# Secure vault configuration
```

Comment:
If this module is being based on latest `wso2base` then these configs are not needed any more.

Reply:
Based on wso2base 1.0.0